### DISCUSSION TITLE
Multi-Threading Processing?

### DISCUSSION THREAD

**Discussion by tyler-corcoran** [casact/chainladder-python#415] [Q&A]
I'm curious if parallel processing within the chainladder package is possible, specifically for use on simulations. I see there's no explicit documentation here, but maybe it's possible through the API dependences?

**Comment by jbogaardt** [comment]
Minimal. I believe [GridSearch](https://chainladder-python.readthedocs.io/en/latest/modules/generated/chainladder.GridSearch.html#:~:text=n_jobs%3A%20int%2C%20default,for%20more%20details.) has the same `n_jobs` parallelization that sklearn's does. but not much else otherwise.
